src/iframes/sign-in-button/sign-in-button.tsx

```diff
@@ -87,7 +89,7 @@ export const SignInButton = () => {
 
   // fetch rooms of first space and send it to space-panel iframe
   const firstSpaceID = spaces?.results[0]?.id
```

Comment:
This could check also for `spaces?.length > 0` 
Since I've seen you are using ramda it could be `head(spaces)` 
I'm giving tips to use ramda. Who would have guessed! Haha! :P

Reply:
Here the `?` after `[0]` serves as a check if length is long enough. You're right, i could use ramda here but i need to protect agains null `spaces`, so it becomes:
```
head(spaces?.results || [])?.id
```
I'm not sure what is easier to read, maybe it's just a personal preference, what do you think?